services/ordering/src/EShopOnAbp.OrderingService.Domain.Shared/Orders/OrderCancelledEto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using Volo.Abp.Domain.Entities.Events.Distributed;
+using Volo.Abp.EventBus;
+
+namespace EShopOnAbp.OrderingService.Orders;
+
+[EventName("EShopOnAbp.Order.Cancelled")]
+public class OrderCancelledEto : EtoBase
+{
+    public Guid OrderId { get; set; }
+    public int OrderNo { get; set; }
+    public DateTime OrderDate { get; set; }
+    public BuyerEto Buyer { get; set; }
+    public List<OrderItemEto> Items { get; set; } = new();
```

Comment:
One of the obvious subscriber to this integration event will be the PaymentService. And it will need PaymentRequestId where we do not expose in here.

This Eto should include `PaymentRequestId` for PaymentService.